### ISSUE TITLE
Upgrade from manager 3.0 to (latest) master fails to upgrade: `No packages marked for update`

### ISSUE THREAD

**Issue by ShlomiBalalis** [scylladb/scylla-manager#3215]
When upgrading from manager 3.0 (3.0.0-0.20220523.5501e5d7f53) to the manager master (3.0-dev-0.20220713.0ca8c73f31c-SNAPSHOT-0ca8c73f31c, latest master at the time of writing) the upgrade command does not upgrade the server/client and it remains in 3.0.0:
Before the upgrade:
```
< t:2022-08-10 16:30:51,188 f:remote_base.py  l:520  c:RemoteLibSSH2CmdRunner p:DEBUG > Running command "sudo sctool version"...
< t:2022-08-10 16:30:51,244 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > Client version: 3.0.0-0.20220523.5501e5d7f53
< t:2022-08-10 16:30:51,245 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > Server version: 3.0.0-0.20220523.5501e5d7f53
< t:2022-08-10 16:34:49,780 f:remote_base.py  l:520  c:RemoteLibSSH2CmdRunner p:DEBUG > Running command "sudo curl -o /etc/yum.repos.d/scylla-manager.repo -L http://downloads.scylladb.com/manager/rpm/unstable/centos/master/latest/scylla-manager.repo"...
```
The upgrade process:
```
< t:2022-08-10 16:34:49,780 f:remote_base.py  l:520  c:RemoteLibSSH2CmdRunner p:DEBUG > Running command "sudo curl -o /etc/yum.repos.d/scylla-manager.repo -L http://downloads.scylladb.com/manager/rpm/unstable/centos/master/latest/scylla-manager.repo"...
< t:2022-08-10 16:34:50,782 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG >                                  Dload  Upload   Total   Spent    Left  Speed
< t:2022-08-10 16:34:51,283 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > ^M  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0^M100   206  100   206    0     0   1059      0 --:--:-- --:--:-- --:--:--  1067
< t:2022-08-10 16:34:51,283 f:base.py         l:140  c:RemoteLibSSH2CmdRunner p:DEBUG > STDERR:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
< t:2022-08-10 16:34:51,283 f:base.py         l:140  c:RemoteLibSSH2CmdRunner p:DEBUG >                                  Dload  Upload   Total   Spent    Left  Speed
< t:2022-08-10 16:34:51,283 f:base.py         l:140  c:RemoteLibSSH2CmdRunner p:DEBUG > 
< t:2022-08-10 16:34:51,283 f:base.py         l:140  c:RemoteLibSSH2CmdRunner p:DEBUG >   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
< t:2022-08-10 16:34:51,283 f:base.py         l:140  c:RemoteLibSSH2CmdRunner p:DEBUG > 100   206  100   206    0     0   1059      0 --:--:-- --:--:-- --:--:--  1067
< t:2022-08-10 16:34:51,283 f:base.py         l:142  c:RemoteLibSSH2CmdRunner p:DEBUG > Command "sudo curl -o /etc/yum.repos.d/scylla-manager.repo -L http://downloads.scylladb.com/manager/rpm/unstable/centos/master/latest/scylla-manager.repo" finished with status 0
< t:2022-08-10 16:34:51,283 f:remote_base.py  l:520  c:RemoteLibSSH2CmdRunner p:DEBUG > Running command "sudo yum update scylla-manager-server scylla-manager-client -y"...
< t:2022-08-10 16:34:51,446 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > Loaded plugins: fastestmirror
< t:2022-08-10 16:34:51,496 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > Loading mirror speeds from cached hostfile
< t:2022-08-10 16:34:51,500 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG >  * base: download.cf.centos.org
< t:2022-08-10 16:34:51,500 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG >  * epel: coresite.mm.fcix.net
< t:2022-08-10 16:34:51,501 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG >  * extras: download.cf.centos.org
< t:2022-08-10 16:34:51,502 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG >  * updates: download.cf.centos.org
< t:2022-08-10 16:34:52,200 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > No packages marked for update
< t:2022-08-10 16:34:52,701 f:base.py         l:142  c:RemoteLibSSH2CmdRunner p:DEBUG > Command "sudo yum update scylla-manager-server scylla-manager-client -y" finished with status 0
```
After the upgrade:
```
< t:2022-08-10 16:36:57,496 f:remote_base.py  l:520  c:RemoteLibSSH2CmdRunner p:DEBUG > Running command "sudo sctool version"...
< t:2022-08-10 16:36:57,554 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > Client version: 3.0.0-0.20220523.5501e5d7f53
< t:2022-08-10 16:36:57,557 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > Server version: 3.0.0-0.20220523.5501e5d7f53
```

It's important to note that, while upgrading from 3.0 fails, upgrading from 2.6 to the latest master has no issues.

Logs:
db-cluster: https://cloudius-jenkins-test.s3.amazonaws.com/48914aa7-fbbd-4995-81ae-e1d6868f4ed1/20220810_163745/db-cluster-48914aa7.tar.gz
loader-set: https://cloudius-jenkins-test.s3.amazonaws.com/48914aa7-fbbd-4995-81ae-e1d6868f4ed1/20220810_163745/loader-set-48914aa7.tar.gz
monitor-set: https://cloudius-jenkins-test.s3.amazonaws.com/48914aa7-fbbd-4995-81ae-e1d6868f4ed1/20220810_163745/monitor-set-48914aa7.tar.gz
sct: https://cloudius-jenkins-test.s3.amazonaws.com/48914aa7-fbbd-4995-81ae-e1d6868f4ed1/20220810_163745/sct-runner-48914aa7.tar.gz
(the manager server log is in the monitor-set, the agent logs are with the db-cluster logs)

**Comment by karol-kokoszka** [comment]
I don't see any clear sign that it's the scylla-manager issue.
After taking a look at 
```
sct: https://cloudius-jenkins-test.s3.amazonaws.com/48914aa7-fbbd-4995-81ae-e1d6868f4ed1/20220810_163745/sct-runner-48914aa7.tar.gz
```
I've found:
```
< t:2022-08-10 16:29:04,164 f:remote_base.py  l:520  c:RemoteLibSSH2CmdRunner p:DEBUG > Running command "sudo curl -o /etc/yum.repos.d/scylla-manager.repo -L http://downloads.scylladb.com.s3.amazonaws.com/rpm/centos/scylladb-manager-3.0.repo"...
< t:2022-08-10 16:29:04,701 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG >   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
< t:2022-08-10 16:29:05,201 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG >                                  Dload  Upload   Total   Spent    Left  Speed
< t:2022-08-10 16:29:05,701 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > 
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   177  100   177    0     0   5752      0 --:--:-- --:--:-- --:--:--  5709
100   177  100   177    0     0   5731      0 --:--:-- --:--:-- --:--:--  5709
< t:2022-08-10 16:29:05,702 f:base.py         l:140  c:RemoteLibSSH2CmdRunner p:DEBUG > STDERR:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
< t:2022-08-10 16:29:05,702 f:base.py         l:140  c:RemoteLibSSH2CmdRunner p:DEBUG >                                  Dload  Upload   Total   Spent    Left  Speed
< t:2022-08-10 16:29:05,702 f:base.py         l:140  c:RemoteLibSSH2CmdRunner p:DEBUG > 
< t:2022-08-10 16:29:05,702 f:base.py         l:140  c:RemoteLibSSH2CmdRunner p:DEBUG >   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
< t:2022-08-10 16:29:05,702 f:base.py         l:140  c:RemoteLibSSH2CmdRunner p:DEBUG > 100   177  100   177    0     0   5752      0 --:--:-- --:--:-- --:--:--  5709
< t:2022-08-10 16:29:05,702 f:base.py         l:140  c:RemoteLibSSH2CmdRunner p:DEBUG > 100   177  100   177    0     0   5731      0 --:--:-- --:--:-- --:--:--  5709
< t:2022-08-10 16:29:05,702 f:base.py         l:142  c:RemoteLibSSH2CmdRunner p:DEBUG > Command "sudo curl -o /etc/yum.repos.d/scylla-manager.repo -L http://downloads.scylladb.com.s3.amazonaws.com/rpm/centos/scylladb-manager-3.0.repo" finished with status 0
< t:2022-08-10 16:29:05,702 f:remote_base.py  l:520  c:RemoteLibSSH2CmdRunner p:DEBUG > Running command "sudo yum install -y scylla-manager-server scylla-manager-client"...
< t:2022-08-10 16:29:05,859 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > Loaded plugins: fastestmirror
< t:2022-08-10 16:29:05,908 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > Loading mirror speeds from cached hostfile
< t:2022-08-10 16:29:05,911 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG >  * base: download.cf.centos.org
< t:2022-08-10 16:29:05,911 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG >  * epel: coresite.mm.fcix.net
< t:2022-08-10 16:29:05,913 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG >  * extras: download.cf.centos.org
< t:2022-08-10 16:29:05,913 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG >  * updates: download.cf.centos.org
< t:2022-08-10 16:29:06,249 f:db_log_reader.py l:113  c:sdcm.db_log_reader   p:DEBUG > 2022-08-10T16:29:06+00:00 manager-upgrade-bump-bas-db-node-48914aa7-2     !INFO | systemd-networkd[439]: eth0: DHCPv6 address 2600:1f18:7f4:3c00:e0c4:3cf9:e105:4acc/128 timeout preferred 140 valid 450
< t:2022-08-10 16:29:06,409 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > Resolving Dependencies
< t:2022-08-10 16:29:06,409 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > --> Running transaction check
< t:2022-08-10 16:29:06,409 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > ---> Package scylla-manager-client.x86_64 0:3.0.0~0.20220523.5501e5d7f53-1 will be installed
< t:2022-08-10 16:29:06,410 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > ---> Package scylla-manager-server.x86_64 0:3.0.0~0.20220523.5501e5d7f53-1 will be installed
< t:2022-08-10 16:29:06,577 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > --> Finished Dependency Resolution
< t:2022-08-10 16:29:06,679 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > 
< t:2022-08-10 16:29:06,679 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > Dependencies Resolved
< t:2022-08-10 16:29:06,680 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > 
< t:2022-08-10 16:29:06,680 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > ================================================================================
< t:2022-08-10 16:29:06,680 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG >  Package       Arch   Version                        Repository            Size
< t:2022-08-10 16:29:06,680 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > ================================================================================
< t:2022-08-10 16:29:06,680 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > Installing:
< t:2022-08-10 16:29:06,680 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG >  scylla-manager-client
< t:2022-08-10 16:29:06,680 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG >                x86_64 3.0.0~0.20220523.5501e5d7f53-1 scylladb-manager-3.0 6.6 M
< t:2022-08-10 16:29:06,680 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG >  scylla-manager-server
< t:2022-08-10 16:29:06,680 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG >                x86_64 3.0.0~0.20220523.5501e5d7f53-1 scylladb-manager-3.0  20 M
< t:2022-08-10 16:29:06,680 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > 
< t:2022-08-10 16:29:06,680 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > Transaction Summary
< t:2022-08-10 16:29:06,680 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > ================================================================================
< t:2022-08-10 16:29:06,680 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > Install  2 Packages
< t:2022-08-10 16:29:06,680 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > 
< t:2022-08-10 16:29:06,680 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > Total download size: 26 M
< t:2022-08-10 16:29:06,681 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > Installed size: 79 M
< t:2022-08-10 16:29:06,681 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > Downloading packages:
< t:2022-08-10 16:29:07,436 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > --------------------------------------------------------------------------------
< t:2022-08-10 16:29:07,436 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > Total                                               35 MB/s |  26 MB  00:00     
< t:2022-08-10 16:29:07,438 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > Running transaction check
< t:2022-08-10 16:29:07,438 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > Running transaction test
< t:2022-08-10 16:29:07,444 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > Transaction test succeeded
< t:2022-08-10 16:29:07,444 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > Running transaction
< t:2022-08-10 16:29:09,653 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > Scylla Manager is up to date: installed=3.0.0-0.20220523.5501e5d7f53
< t:2022-08-10 16:29:10,128 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG >   Verifying  : scylla-manager-client-3.0.0~0.20220523.5501e5d7f53-1.x86_6   1/2 
< t:2022-08-10 16:29:10,197 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG >   Verifying  : scylla-manager-server-3.0.0~0.20220523.5501e5d7f53-1.x86_6   2/2 
< t:2022-08-10 16:29:10,197 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > 
< t:2022-08-10 16:29:10,197 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > Installed:
< t:2022-08-10 16:29:10,197 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG >   scylla-manager-client.x86_64 0:3.0.0~0.20220523.5501e5d7f53-1                 
< t:2022-08-10 16:29:10,197 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG >   scylla-manager-server.x86_64 0:3.0.0~0.20220523.5501e5d7f53-1                 
< t:2022-08-10 16:29:10,197 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > 
< t:2022-08-10 16:29:10,197 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > Complete!
< t:2022-08-10 16:29:10,698 f:base.py         l:142  c:RemoteLibSSH2CmdRunner p:DEBUG > Command "sudo yum install -y scylla-manager-server scylla-manager-client" finished with status 0
```

Command output says that:
- you copied scylla-manager.repo from our location to `/etc/yum.repos.d/scylla-manager.repo`
- just after the copy action, upgrade scylla-manager.server and client to the latest versions

Yum found out that according to his knowledge the latest version is `Scylla Manager is up to date: installed=3.0.0-0.20220523.5501e5d7f53`
So exactly the same that was installed before.

Content of `scylla-manager.repo` already contains an information about which version (timestamp) you want to install, different build are split per timestamp directory:
```
[scylla-manager]
name=Scylla for Centos $releasever - $basearch
baseurl=http://downloads.scylladb.com/manager/rpm/unstable/centos/master/2022-09-21T21:54:50Z/scylla-manager/$basearch/
enabled=1
gpgcheck=0
```

There is a chance that there is etckeeper on the machine that provides VCS to the configuration.
In scylla-cloud, we ensure that etc i commit by executing 
`date --rfc-3339=ns | sudo tee /etc/.touch && sudo etckeeper commit "`+commit+`"`
after we change the content of repo files.

@ShlomiBalalis can you check if this is the case ? Becasue it really looks that according to YUM, the latest version is installed (even tough the latest = current one)

**Comment by fgelcer** [comment]
@karol-kokoszka , last master version of Scylla manager is from May 23rd? Does that make sense? aren't there at least weekly builds?

**Comment by karol-kokoszka** [comment]
@fgelcer 
> last master version of Scylla manager is from May 23rd? Does that make sense? aren't there at least weekly builds?
No, there is no information that latest master version of Scylla manager is from 23rd of May.

Scylla Manager from 23rd of May was already installed on the node and the same version was in `/etc/yum.repos.d/scylla-manager.repo` which basically keeps an information about Scylla Manager rpm repository.
Later, test just replaced that file with the new version that most likely kept the correct timestamp (not May 23rd).

I wrote 
> There is a chance that there is etckeeper on the machine that provides VCS to the configuration.
In scylla-cloud, we ensure that etc is commit by executing
date --rfc-3339=ns | sudo tee /etc/.touch && sudo etckeeper commit "+commit+"
after we change the content of repo files.

And I wrote it because if there is etckeeper, then it's not enough to just copy new file into `/etc`, BUT you need to commit it to VCS as well.
And this is exactly what we do in Scylla-Cloud when we upgrade Manager to the newer version, as just copying the repo definition file is not always enough.

**Comment by karol-kokoszka** [comment]
if it's not etckeeper then consider this article https://www.hivelocity.net/kb/how-to-clear-the-yum-cache/

**Comment by karol-kokoszka** [comment]
@fgelcer @ShlomiBalalis I checked it once again.
Upgrade works fine on UBUNTU (according to the logs attached):
```
2022-08-10T16:35:18+00:00 manager-upgrade-bump-bas-db-node-48914aa7-1   !NOTICE | sudo[13237]: scyllaadm : TTY=unknown ; PWD=/home/scyllaadm ; USER=root ; COMMAND=/usr/bin/apt-get install -y scylla-manager-agent
2022-08-10T16:35:23+00:00 manager-upgrade-bump-bas-db-node-48914aa7-1     !INFO | scylla-manager-agent[13438]: {"L":"INFO","T":"2022-08-10T16:35:23.605Z","M":"Scylla Manager Agent","version":"3.0-dev-0.20220713.0ca8c73f31c-SNAPSHOT-0ca8c73f31c","pid":13438,"_trace_id":"IQKW6V2KQA-nm8qOeqGGyg"}
```

It doesn't work on CENTOS.
If the latest version that is currently install is <= current one, yum will keep the current one.

I tested it on CENTOS running locally in docker container.
```
docker pull centos
docker run -i -t centos
```
then
```
[root@81207a77ac9d yum.repos.d]# curl -o /etc/yum.repos.d/scylla-manager.repo -L http://downloads.scylladb.com/rpm/centos/scylladb-manager-3.0.repo
yum update
yum install scylla-manager-server
---- at this point scylla-manager-server-3.0.2 is installed
---- then
curl -o /etc/yum.repos.d/scylla-manager.repo -L http://downloads.scylladb.com/manager/rpm/unstable/centos/master/latest/scylla-manager.repo
yum update
---- try to force specific package to update
[root@81207a77ac9d yum.repos.d]# yum update scylla-manager-server
Last metadata expiration check: 0:00:11 ago on Wed 07 Dec 2022 05:36:55 PM UTC.
Dependencies resolved.
Nothing to do.
Complete!
```
Scylla-manager is still in 3.0.2 version.
YUM sees the newest version that is installed already and wont downgrade to 3.0.0~dev.

To work it around on the same environment (3.0.2 is install, repo of 3.0.0~dev is kept in /etc/yum.repos.d):
```
yum remove scylla-manager-server

[root@81207a77ac9d yum.repos.d]# yum install scylla-manager-server
Last metadata expiration check: 0:02:45 ago on Wed 07 Dec 2022 05:36:55 PM UTC.
Dependencies resolved.
===================================================================================================================================================================================================================
 Package                                         Architecture                     Version                                                                           Repository                                Size
===================================================================================================================================================================================================================
Installing:
 scylla-manager-server                           x86_64                           3.0.0~dev-0.20221206.e9f44920539-SNAPSHOT-e9f44920539-1                           scylla-manager                            21 M

Transaction Summary
===================================================================================================================================================================================================================
Install  1 Package

Total download size: 21 M
Installed size: 65 M
Is this ok [y/N]: y
Downloading Packages:
scylla-manager-server-3.0-dev-0.20221206.e9f44920539-SNAPSHOT-e9f44920539_linux.x86_64.rpm                                                                                          11 MB/s |  21 MB     00:01    
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Total                                                                                                                                                                               11 MB/s |  21 MB     00:01     
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                                                                                                                                                           1/1 
  Running scriptlet: scylla-manager-server-3.0.0~dev-0.20221206.e9f44920539-SNAPSHOT-e9f44920539-1.x86_64                                                                                                      1/1 
scylla-manager:x:1000:
scylla-manager:x:997:1000::/home/scylla-manager:/usr/sbin/nologin

  Installing       : scylla-manager-server-3.0.0~dev-0.20221206.e9f44920539-SNAPSHOT-e9f44920539-1.x86_64                                                                                                      1/1 
  Running scriptlet: scylla-manager-server-3.0.0~dev-0.20221206.e9f44920539-SNAPSHOT-e9f44920539-1.x86_64                                                                                                      1/1 
Scylla Manager is up to date: installed=3.0-dev-0.20221206.e9f44920539-SNAPSHOT-e9f44920539

  Verifying        : scylla-manager-server-3.0.0~dev-0.20221206.e9f44920539-SNAPSHOT-e9f44920539-1.x86_64                                                                                                      1/1 

Installed:
  scylla-manager-server-3.0.0~dev-0.20221206.e9f44920539-SNAPSHOT-e9f44920539-1.x86_64                                                                                                                             

Complete!
```

To sum it up:
- Remove newer (according to YUM ofc) version of scylla-manager-server package with cmd `yum remove scylla-manager-server`
- Install master-dev version `yum install scylla-manager-server`

Pls confirm if the problem is solved.

**Comment by ShlomiBalalis** [comment]
While the remove solution is solid, I'd rather stick to the documented process. I opened an issue for releng to update the master builds' version number to 3.1-dev: https://github.com/scylladb/scylla-pkg/issues/3209
